### ISSUE TITLE
Creating private_jwk fails in AWS lambda function

### ISSUE THREAD

**Issue by kk738** [guillp/jwskate#15]
- `jwskate` version: 0.8.0
- Python version: 3.7
- Operating System: Amazon Linux 2 (Lambda function)

### Description

I installed the library as a layer and attached it to my lambda function in AWS. 
Basically by:

```bash
mkdir lambda_layers
cd lambda_layers
mkdir python
cd python
pip install jwskate -t ./
cd ..
zip-r python_modules.zip .
```

Some stuff is working, but some things are very weird, e.g. executing

```python
from jwskate import Jwk

private_jwk = (
    Jwk.generate(alg="ES256")  # select the signature or encryption alg here
    .with_kid_thumbprint()  # optionally, include a RFC7638 compliant thumbprint as kid
    .with_usage_parameters()  # optionally, include 'use' and 'key_ops'
)
```

results in:

```json
{
  "errorMessage": "",
  "errorType": "NotImplementedError",
  "stackTrace": [
    "  File \"/var/task/lambda_function.py\", line 11, in lambda_handler\n    Jwk.generate(alg=\"ES256\")  # select the signature or encryption alg here\n",
    "  File \"/opt/python/jwskate/jwk/base.py\", line 1044, in generate\n    raise NotImplementedError\n"
  ]
}
```

Furthermore I get "type object 'Jwk' has no attribute 'from_pem'" for 

```python
private_jwk_from_file = (
        Jwk.from_pem(key, password="Th1s_P@ssW0rD_iS_5o_5tr0nG!")
        .with_usage_parameters(alg="ES256")  # adds back the alg, use and key_ops parameters
        .with_kid_thumbprint()  # adds back the thumbprint as kid
    )
```

Is there something I did wrong?

**Comment by guillp** [comment]
I don't know anything about AWS, but given that `Jwk.generate` is unimplemented and that `"type object 'Jwk' has no attribute 'from_pem'`, I'd say you probably obtained a version of `jwskate` that is older than 0.7.

Please confirm which version you have.

**Comment by kk738** [comment]
Thanks. I think the issue was, that on the linux machine I created the bundle to upload it to AWS, Python 3.7 was installed, that's why pip chose V0.6 automatically I guess.

Never mind, I reinstalled all things on an updated machine and had to do some adjustment for cryptography I found here https://github.com/pyca/cryptography/issues/6391 and it works fine now.

So for anybody who wants to install this in lambda use:
```bash
mkdir lambda_layers
cd lambda_layers
mkdir python
cd python
pip install packageName --platform manylinux2010_x86_64 --implementation cp --only-binary=:all: --upgrade -t ./
cd ..
zip -r python_modules.zip .
```
